pkgs/by-name/ka/kanha/package.nix

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Did you format your file with `nixfmt-rfc-style` ?

Reply:
based on the "peculiarities" I've noticed that the formatter has, it seems they did.